pkg/test/resourcefixture/testdata/basic/apigee/v1alpha1/apigeeenvgroup/apigeeenvgroup-basic/_http.log

```diff
@@ -56,21 +55,53 @@ X-Xss-Protection: 0
   "name": "organizations/${projectId}/operations/${operationID}",
   "response": {
     "@type": "type.googleapis.com/google.cloud.apigee.v1.GoogleCloudApigeeV1EnvironmentGroup",
-    "createdAt": "2024-04-01T12:34:56.123456Z",
```

Comment:
Can we improve the mock test to include this field (and `lastModifiedAt` below)? The KCC API uses *int64 to hold these two fields, we want to make sure it is correctly mapped in status.